apps/betterangels-backend/hmis/schema.py

```diff
@@ -63,12 +69,35 @@ def hmis_note(self, info: Info, id: ID) -> HmisNoteType:
             note_hmis_id=hmis_note.hmis_id,
         )
 
+        client_program = (
+            _get_client_program(program_data) if (program_data := note_data.pop("client_program", None)) else None
+        )
+
         hmis_note = resolvers.update(info, hmis_note, {**note_data})
 
+        hmis_note.client_program = client_program  # type: ignore
+
         return cast(HmisNoteType, hmis_note)
 
     hmis_notes: OffsetPaginated[HmisNoteType] = strawberry_django.offset_paginated(permission_classes=[IsAuthenticated])
 
+    @strawberry.field()
+    def hmis_client_programs(
+        self,
+        info: Info,
+        client_id: ID,
+    ) -> list[HmisClientProgramType]:
+        hmis_api_bridge = HmisRestApiBridge(info=info)
+
+        client_hmis_id = HmisClientProfile.objects.get(pk=client_id).hmis_id
```

Comment:
Get can throw an exception where `filter` + `values_list` can't.  Up to you if you want to adopt this style.

```python
client_hmis_id = HmisClientProfile.objects.filter(pk=client_id).values_list("hmis_id", flat=True).first()
```